src/libraries/System.IO.FileSystem/tests/File/Append.cs

```diff
@@ -77,4 +77,23 @@ public void NullEncoding()
             Assert.Throws<ArgumentNullException>(() => File.AppendAllLines(GetTestFilePath(), new string[] { "Text" }, null));
         }
     }
+
+    public class File_AppendAllBytes: File_ReadWriteAllBytes
```

Comment:
I just remembered that I needed to add null params tests because I focused on following the rules and coding style. But wait, where is the AppendAllText or AppendAllLines invalid param tests?!

Reply:
In [File_ReadWriteAllText](https://github.com/MojtabaTajik/runtime/blob/f4b06109929b96ea570c38ef912f2a851985aa86/src/libraries/System.IO.FileSystem/tests/File/ReadWriteAllText.cs#L36).